Microsoft.Toolkit.Graph.Controls/Controls/PeoplePicker/PeoplePicker.xaml

```diff
@@ -2,36 +2,34 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" 
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:local="using:Microsoft.Toolkit.Graph.Controls"
-    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+    xmlns:ex="using:Microsoft.Toolkit.Uwp.UI.Extensions">
+    <ResourceDictionary.MergedDictionaries>
+        <ResourceDictionary Source="ms-appx:///Microsoft.Toolkit.Uwp.UI.Controls/TokenizingTextBox/TokenizingTextBox.xaml" />
```

Comment:
Why do we need to reference this here? Shouldn't it be automatic?

Reply:
You would think, but nope. ☹

The search for resources only appears to go from our assembly to the generic.xaml. (see wpf behavior (similar) [here](https://docs.microsoft.com/en-us/dotnet/desktop-wpf/fundamentals/xaml-resources-define#static-resource-lookup-behavior) and [here](https://docs.microsoft.com/en-us/dotnet/desktop-wpf/fundamentals/xaml-resources-define#dynamic-resource-lookup-behavior)) - At least that's what I guess is happening.